### ISSUE TITLE
A new album is created for every photo uploaded

### ISSUE THREAD

**Issue by Gatsby-PF** [moxi9/phpfox#782]
Photos Module:
When uploading pics to an album (using the pencil icon within the album), a new album gets created for each new photo uploaded. So I end up with a column of several albums with several photos whereas one album was intended.  Upon opening any album that was created this way, I see all photos that were uploaded. Upon deleting any one album, all albums get deleted.

**Comment by Data66PF** [comment]
I cannot reproduce this in 4.1.1

**Comment by Gatsby-PF** [comment]
I think I determined the issue. 

When selecting "Upload Photo" button to upper right, a dialog opens. I select the album first, then upload a photo. When photo uploads, I get a second different dialog that prompts for name edit and category select.  As long as I do not use the "move photo to album" select at the top of this 2nd dialog, subsequent photo uploads load into the same album, repeating the above process each time. This was done successfully three times, maintaining just one album. For the fourth photo, I did the same, but made one change.  This time I selected "move photo to album" selecting the same album name of course, and then accepted the photo.  This time I now could see two albums by the same name, each with all four photos in them, and deleting one album, deleted them all.

The issue has to do with using the move album select in the second upload dialog, which I think is completely unnecessary.
